app/src/main/AndroidManifest.xml

```diff
@@ -4,13 +4,19 @@
     package="com.fpvout.digiview">
 
     <uses-feature android:name="android.hardware.usb.host" />
+    <uses-permission android:name="android.permission.WRITE_EXTERNAL_STORAGE" />
+    <uses-permission android:name="android.permission.RECORD_AUDIO"/>
+    <uses-permission android:name="android.permission.CAMERA"/>
+    <uses-permission android:name="android.permission.READ_EXTERNAL_STORAGE" />
     <uses-permission android:name="android.permission.WAKE_LOCK" />
 
     <application
         android:allowBackup="false"
         android:icon="@mipmap/ic_launcher"
         android:label="@string/app_name"
         android:roundIcon="@mipmap/ic_launcher_round"
+        android:largeHeap="true"
+        android:requestLegacyExternalStorage="true"
```

Comment:
Would be best to use scoped storage and shared directories :
https://developer.android.com/training/data-storage/shared/media

Reply:
mhh i know bad habit to us the old implementation ;-)

Reply:
for backwards compatibility I leave it as it is. we can change it later.

Reply:
ok good to know! i will update that